common/playlist.h

```diff
@@ -43,11 +43,9 @@ struct playlist_entry {
     // Used for unshuffling: the pl_index before it was shuffled. -1 => unknown.
     int original_index;
 
-    // Set to true if playback didn't seem to work, or if the file could be
-    // played only for a very short time. This is used to make playlist
-    // navigation just work in case the user has unplayable files in the
-    // playlist.
-    bool playback_short : 1;
+    // Set to true if we're trying to go backwards in the playlist.
```

Comment:
on the current entry? on the to-be-tried entry?
what happens if it's set?
(the comment should ideally answer this)

Reply:
It was a bit terse. Hopefully better now.